pkgs/development/python-modules/devolo-home-control-api/default.nix

```diff
@@ -36,7 +36,10 @@ buildPythonPackage rec {
     # setup.py is not able to detect the version with setuptools_scm
     substituteInPlace setup.py \
       --replace "setuptools_scm" "" \
-      --replace 'use_scm_version=True' 'use_scm_version="${version}"'
+      --replace 'use_scm_version=True' 'version="${version}"'
+
+    substituteInPlace devolo_home_control_api/__init__.py \
+      --replace '"0.0.0"' '"${version}"'
```

Comment:
A lot of these which use `setuptools-scm` could just use `SETUPTOOLS_SCM_PRETEND_VERSION`. And then they are more resilient to changes in implementation.

Reply:
yeah, changed that.